data/tools/wmlindent

```diff
@@ -306,7 +306,7 @@ def sigint_handler(signal, frame):
     This overrides Python's default behavior of displaying a traceback when ctrl-c is pressed.
     """
     print ('Aborted by pressing ctrl-c')
-    sys.exit(0)
+    sys.exit(1)
```

Comment:
The comments above should stay in sync with the code.

Reply:
> The comments above should stay in sync with the code.

Not sure what you are referring to. Does "comments" refer to the function docstring or to the print message? I don't quite understand the request actually, could you rephrase it?

Reply:
Yes, the docstring spells out what is done... `a one-line message is displayed and Python exits with Status 0`

Reply:
resolved in https://github.com/wesnoth/wesnoth/pull/6500/commits/42faf4f1ab3717e7364efc94b36f93c28fed7dfb